### DISCUSSION TITLE
Mega SG adapters

### DISCUSSION THREAD

**Discussion by douglie007** [sanni/cartreader#1032] [Ideas & Suggestions]
Could it be programmed to use the mega SG adapters for the Game Gear, and sg1000 throught the genesis slot?

**Comment by andy-miles** [comment]
It works.  I have had success using the Analogue adapters for GG and JPN Mark III games via the genesis slot using the "Retron3in1" configuration.  See https://github.com/sanni/cartreader/issues/480#issuecomment-2327063850
